frontend/src/components/app/app.tsx

```diff
@@ -63,10 +63,18 @@ const App: FC = () => {
           }
         />
         <Route
-          path={AppRoute.UAM_CREATE_GROUP}
+          path={AppRoute.UAM_CONFIGURATE_GROUP}
           element={
             <AuthorizedWrapper>
-              <UAMGroupsCreate />
+              <UAMConfigurateGroup mode="create" name="Create" />
+            </AuthorizedWrapper>
+          }
+        />
+        <Route
+          path={AppRoute.UAM_CONFIGURATE_GROUP_EDIT}
+          element={
+            <AuthorizedWrapper>
+              <UAMConfigurateGroup mode="edit" name="Edit" />
```

Comment:
```suggestion
              <UAMConfigurateGroup />
            </AuthorizedWrapper>
          }
        />
        <Route
          path={AppRoute.UAM_CONFIGURATE_GROUP_EDIT}
          element={
            <AuthorizedWrapper>
              <UAMConfigurateGroup />
```
handle it by `const isEditMode = Boolean(id)`